test/metabase/driver/generic_sql_test.clj

```diff
@@ -18,7 +18,7 @@
 (def ^:private generic-sql-engines
   (delay (set (for [engine datasets/all-valid-engines
                     :let   [driver (driver/engine->driver engine)]
-                    :when  (not (contains? #{:bigquery :presto} engine))                 ; bigquery and presto don't use the generic sql implementations of things like `field-avg-length`
+                    :when  (not (contains? #{:bigquery :presto :sparksql} engine))       ; bigquery and presto don't use the generic sql implementations of things like `field-avg-length`
```

Comment:
if you're going to change the code here you should update the comment as well. Something like `bigquery, presto, etc. don't use...` (changing just the beginning) would suffice

https://github.com/metabase/metabase/wiki/Metabase-Clojure-Style-Guide#make-sure-to-update-comments

Reply:
Done